### DISCUSSION TITLE
when will Realtime database web support  come ?

### DISCUSSION THREAD

**Discussion by sabreys** [firebase/flutterfire#5201] [Feature request]
when will Realtime database web support  come ? I am waiting that for chat app but I don't want to use firestore because of  pricing.

**Comment by Papajik** [comment]
I have the same question.  I found some mentions that web support for RTDB will be early this year but nothing more on this topic. There is also no mention in the roadmap about RTDB. Is RTDB really used so little that adding the web suport doesn't seem too appealing?

I am using ESP32 in my project and there is no support for Firestore streaming and RTDB streaming is much better choice than requesting all data every few seconds to check if there is any update due to data consumption.
